src/classes/public/Context/GitHubContext/InstallationGitHubContext.ps1

```diff
@@ -43,7 +43,7 @@
         $this.HttpVersion = $Object.HttpVersion
         $this.PerPage = $Object.PerPage
         $this.ClientID = $Object.ClientID
-        $this.TokenExpirationDate = $Object.TokenExpirationDate
+        $this.TokenExpiresAt = if ($Object.TokenExpiresAt) { $Object.TokenExpiresAt } else { $Object.TokenExpirationDate }
```

Comment:
Dont use all the mappings. Lets only focus on the one type, as its the one we control from the function that creates the functions.